_i18n/en/_posts/2017-09-14-ruby-on-rails-support.md

```diff
@@ -1,43 +1,43 @@
 ---
 layout: post
-title: Localize your Ruby on Rails app with GitLocalize
+title: Localize Your Ruby on Rails App with GitLocalize
 languages:
   - ja
 image: gitlocalize.png
 ---
 
-Are you struggling on managing i18n YAML files and its localization work with Ruby on Rails? GitLocalize is now here to help you all the tedious i18n work with Rails!
+Struggling to manage your I18n YAML files and the accompanying localization work with Ruby on Rails? Well, we have great news for you. GitLocalize is now here to help you take care of all that tedious Rails I18n work!
 
 # Get Started
 
-If you haven't signed up yet, sign up to GitLocalize for free and get ready to localize your Rails app with zero stress.
+If you haven't signed up yet, register with GitLocalize for free and get ready to localize your Rails app with zero stress.
 
 <a class="btn btn-hero" href="https://gitlocalize.com/auth/github">
   <i class="fa fa-github"></i><span style="margin-left: 6px;">Get Started for Free</span>
 </a>
 
-If you want to learn more on how GitLocalize works, read [our blog article](/posts/introducing-gitlocalize.html).
+Read [our blog article](/posts/introducing-gitlocalize.html) to learn more about how GitLocalize works and what it can do for you.
 
 # Register Your Rails App
 
-If you navigate to `Add Repository` page from your profile page, you will see the form to register your repository on GitHub to GitLoalize platform.
+Navigate to the `Add Repository` button from your profile page. Here, you'll see the form to register your GitHub repository with the GitLocalize platform.
 
-In the middle of the page, you need to specify where your source i18n files are located and where you want to generate translated files to.
+In the middle of the page, specify where your source I18n files are located and where you want to generate translated files.
 
-For example, if you have source English i18n YAML file at `config/locales/en.yml` and want to have translated ones as `config/locales/ja.yml` or `config/locales/es.yml`, set `File` on the right and `config/locales/en.yml` for `Source Path` and `config/locales/%lang%.yml` for `Translation Path`.
+For example, let's say that you have source English I18n YAML files at `config/locales/en.yml` and want to have translated ones at `config/locales/%lang%.yml`. Set the drop-down menu on the left to `File`, then select `config/locales/en.yml` for `Source Path` and `config/locales/%lang%.yml` for `Translation Path`.
```

Comment:
Here, "want to have translated ones at `config/locales/%lang%.yml`", I want to give actual file paths like "config/locales/ja.yml" or "config/locales/en.yml" to explain how this rule works as I wrote first because this `%lang%` might not be understandable for users at first place.

Reply:
That is understandable. I honestly felt like your explanation under of '%lang%' was really good to the point that we could get away with using '%lang%' in the top part, but there is nothing wrong with making the example simpler. I've changed this one back.